src/jit/gentree.h

```diff
@@ -2334,10 +2334,85 @@ struct GenTreeColon: public GenTreeOp
         {}
 };
 
-/* gtCall   -- method call      (GT_CALL) */
+// gtCall   -- method call      (GT_CALL)
 typedef class fgArgInfo *  fgArgInfoPtr;
 enum class InlineObservation;
 
+// Multi-reg return type descriptor of a GT_CALL node.
+// x64 Unix, Arm64, Arm32 and x86 allow a value to be returned in multiple
+// registers. For such calls this struct provides the following info 
+// on their return type
+//    - type of value returned in each return register
+//    - return register numbers in which the value is returned
+//    - a spill mask for lsra/codegen purpose
+//
+// Right now it is implemented to meet the needs of x64 unix.
+// Eventually it will be updated to meet the needs of Arm32/arm64.
```

Comment:
I am not suggesting to express it differently for x86.  Because gtType of TYP_LONG says it all for a call node regarding its return type.  We might as well need a return desc for x86 when we add more state to return desc (e.g. spill mask).  I will update the comment.